src/Aspire.Dashboard/Components/App.razor

```diff
@@ -14,7 +14,22 @@
     <!-- Make the body background dark if dark mode will be enabled to prevent a flash of white
          before fully rendered. This value is from --neutral-fill-layer-rest, but that custom
          property isn't available at this point. -->
-    <style>@@media (prefers-color-scheme: dark) { body { background-color: #272727; } }</style>
+    <style>
+        @@media (prefers-color-scheme: dark) {
+            body {
+                background-color: #272727;
+            }
+        }
+
+        #components-reconnect-modal :is(h5, p) {
+            color: var(--error-ui-foreground-color);
+        }
+
+        #components-reconnect-modal :is(a) {
```

Comment:
```suggestion
        #components-reconnect-modal a {
```

Reply:
I originally had the selector as this, and it did not work. I didn't know why.

Reply:
Hmm ok. The doc I read on `:is` said these styles were equivalent and `:is` is shorthand when there are multiple rules.

Reply:
It seems to be working for me:
![image](https://github.com/dotnet/aspire/assets/9613109/82fb8929-6a78-45c8-b8c4-b74c5dcdc63c)
Maybe a caching glitch or something?

Reply:
Interesting, that's probably what happened... changed, thansk for checking that